scripts/workspace_deploy.py

```diff
@@ -52,8 +52,8 @@ def main(argv):
 
     # workspace info
     try:
-        workspaceFilePath = os.path.join(getattr(config, 'common_outputs_directory'), getattr(config, 'common_outputs_workspace'))
-        with openFile(workspaceFilePath, 'r') as workspaceFile:
+        workspaceFilePath = os.path.join(getRequiredParameter(config, 'common_outputs_directory'), getRequiredParameter(config, 'common_outputs_workspace'))
+        with openFile(workspaceFilePath, 'r') as workspaceFile
```

Comment:
The colon (:) is essential.

Reply:
Thank you, but I have already fix that 4 mins before you. :)